packages/sdk/src/interfaces/types.ts

```diff
@@ -181,7 +189,7 @@ export type MessageLike =
 /**
  * Stuff that can be coerced into a provider.
  */
-export type ProviderLike = string | Provider
+export type ProviderLike = string | Provider | any
```

Comment:
Or `any`? Seems like it breaks the type

Reply:
Agh this is because ethers is a real piece of work sometimes and different ethers versions will appear to be incompatible just because there's some tiny typing difference in a private member somewhere down the chain of inheritance. So I figured "any" was good enough and we could rely on the Provider.isProvider check. I left the original type in case I figure out a way to fix this in the future.